### ISSUE TITLE
Google Colab output files location

### ISSUE THREAD

**Issue by kubinka0505** [facebookresearch/audiocraft#45]
In Google Colab, where are the output files saved? I've looked through `/content`, `/usr/local/lib/python3.10/dist-packages/audiocraft` and `/tmp` with no results.

**Comment by BulatGazizov** [comment]
Same problem running locally - cannot find generated file.
Please help.

**Comment by kubinka0505** [comment]
> Found it in C:\Users{user}\AppData\Local\Temp in case of local run

Looks like it uses `TempFile` 🤷‍♂️

https://github.com/facebookresearch/audiocraft/blob/c9179f87d9f560c96cf6ec4de65a8fcf374c91c5/app.py#L101

**Comment by SatyaDewangan05** [comment]
In colab, output/generated audio is stored in the variable `res`. You can view it but can't download it. You can download it by modifying the `audiocraft.utils.notebook.display_audio()` function by replacing


    for audio in samples:
            ipd.display(ipd.Audio(audio, rate=sample_rate))

by

    count = 0
    for audio in samples:
        count += 1
        file_name_mp3 = f'music_00{count}.mp3' # audio file_name

        audio = ipd.Audio(audio, rate=sample_rate) # display audio in HTML
        ipd.display(audio)
        
        audio = AudioSegment(audio.data, frame_rate=22050, sample_width=2, channels=1)
        audio.export(file_name_mp3, format="mp3", bitrate="64k") # save audio

Or you can write a new function in the new code block also

    import locale
    locale.getpreferredencoding = lambda: "UTF-8"
    !pip install PyDub
<br>

    from pydub import AudioSegment
    try:
        import IPython.display as ipd  # type: ignore
    except ImportError:
        # Note in a notebook...
        pass

    import torch

    def display_audio(samples: torch.Tensor, sample_rate: int):
        """Renders an audio player for the given audio samples.
    
        Args:
            samples (torch.Tensor): a Tensor of decoded audio samples
                with shapes [B, C, T] or [C, T]
            sample_rate (int): sample rate audio should be displayed with.
        """
        assert samples.dim() == 2 or samples.dim() == 3
    
        samples = samples.detach().cpu()
        if samples.dim() == 2:
            samples = samples[None, ...]
        count = 0
        for audio in samples:
            count += 1
            file_name_mp3 = f'music_00{count}.mp3' # audio file_name
    
            audio = ipd.Audio(audio, rate=sample_rate) # display audio in HTML
            ipd.display(audio)
            
            audio = AudioSegment(audio.data, frame_rate=22050, sample_width=2, channels=1)
            audio.export(file_name_mp3, format="mp3", bitrate="64k") # save audio
<br>
Call the `display_audio` function to display the audio player and download the generated audio

    display_audio(res, 32000)

**Comment by samultra99** [comment]
You are incredible! Thank you
